www/templates/diary/place_list_item.html

```diff
@@ -4,7 +4,7 @@
         <!-- Start Time Tag -->
         <!-- <div class="start-time-tag">{{trip.display_end_time}}</div> -->
         <div class="diary-card short" style="padding-top: 10px;">
-            <div style="width: 100%; margin: auto; padding: 5px">
```

Comment:
Why was this padding removed?
We can do without it - but if we do remove it, then we have to update the card height values in `infinite_scroll_list.js > getTripHeight()`

Reply:
Reverting. @sebastianbarry please address this comment on Monday